torch/jit/_script.py

```diff
@@ -53,6 +53,57 @@
     def Attribute(value, type):  # type: ignore
         return value
 
+Attribute.__doc__ = """
+    This method is a pass-through function that returns `value`. It is mostly
```

Comment:
We should add that `torch.jit.Attribute` is only used to annotate types for rhs expression in assignments to attributes in `__init__` of `torch.nn.Module` subtypes

Reply:
Done.